### DISCUSSION TITLE
Following

### DISCUSSION THREAD

**Discussion by emagic2** [community/community#140095] [GitHub Education] [answered]
### Hi everyone,

How I can see the accounts who I followed him and he followed me.

**Comment by masterwww1** [comment]
Hi, piotrsupport991.
I think GitHub doesn’t directly offer a feature to filter accounts that both follow you and you follow back.

**Reply by emagic2** [reply]
Then there is no way I can check the followers who unfollowed me?

**Reply by masterwww1** [reply, marked as answer]
I see what you mean.
There is no way to list the followers unfollowed you.
There is only way that:
- Going to your "Following" list (github.com/your-username?tab=following).
- Visiting each account's profile and checking if they also follow you.
